neural_modelling/src/common/neuron-typedefs.h

```diff
@@ -25,22 +30,13 @@
 #include <common-typedefs.h>
 #include "maths-util.h"
 
-// Determine the type of a spike
-/*
- * defines a spike with either a pay load or not and implements the
- * functionality to extract the key and pay load in both cases. If the
- * spike is compiled as not having a pay load, the pay load will always be
- * returned as 0
- */
-#ifndef __SPIKE_T__
```

Comment:
Don't need this. The file already has a guard and only this file defines the type of spikes.